### PR TITLE
Add HIP-Clang headers for resolving undeclared inclusions

### PR DESCRIPTION
**Pull request by yxsamliu**

### CODE REVIEW

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by sunway513** [comment]
jenkins: ok to test

File: third_party/gpus/rocm_configure.bzl [hunk 1]
```diff
@@ -173,6 +173,10 @@ def _rocm_include_path(repository_ctx, rocm_config):
     inc_dirs.append("/opt/rocm/include/hip/hcc_detail")
     inc_dirs.append("/opt/rocm/hip/include")
 
+    # Add HIP-Clang headers
+    inc_dirs.append("/opt/rocm/llvm/lib/clang/8.0/include")
```
Comment (whchung) [inline third_party/gpus/rocm_configure.bzl]: 8.0.0?

Comment (yxsamliu) [reply third_party/gpus/rocm_configure.bzl]: This is for hip-clang built with HIP/VDI runtime. Its version is 8.0 for some reason.

**Pull request merged**